source/common/ratelimit/ratelimit.proto

```diff
@@ -2,8 +2,6 @@ syntax = "proto3";
 
 option go_package = "ratelimit";
 
-option cc_generic_services = true;
```

Comment:
It might be out of scope for this change, but if possible can we pull this proto from envoy-api and delete this file also? There is a GH open on that.